accounts/graph.py

```diff
@@ -0,0 +1,79 @@
+import json
+import requests
+import msal
+
+from django.contrib.auth import get_user_model
+
+#!! Change this to load parameters from env
+config = json.load(open("parameters.json"))
```

Comment:
This should probably run through the django config system -- lets us set a default in `config.py`

Reply:
Yes, parameters.json (the file itself isn't included in this PR) is what contained things like the tenant info, app ID, and client secret. Those should be loaded through whatever config/env file is used for those type of things on the DB server.